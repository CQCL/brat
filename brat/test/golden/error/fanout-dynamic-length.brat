f(n :: #, Vec(Qubit, n)) -> { Qubit, Qubit }
f(n) = { [/\] }
