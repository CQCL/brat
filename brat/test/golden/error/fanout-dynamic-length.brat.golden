Error in test/golden/error/fanout-dynamic-length.brat on line 2:
f(n) = { [/\] }
         ^^^^

  Type error: Can't fanout a Vec with non-constant length: VPar Ex checking_check_defs_1_f_f.box_2_lambda_fake_source 0

